packages/App/src/balance/statistic/CategoryChart.tsx

```diff
@@ -0,0 +1,166 @@
+import React, {FC} from 'react';
+import {View, Text, StyleSheet} from 'react-native';
+import {useSelector} from 'react-redux';
+import {walletItems} from '../../store/selectors/walletItems';
+import {VictoryPie} from 'victory-native';
+
+enum TransactionType {
+  income = 'income',
+  expenses = 'expenses',
+}
+
+enum Category {
+  Car = 'Car',
+  Health = 'Health',
+  Unknown = 'Unknown',
+  Grocery = 'Grocery',
+  Shopping = 'Shopping',
+  Restaurant = 'Restaurant',
+}
+
+interface Props {
+  month: number;
+}
+
+const CategoryChart: FC<Props> = ({month}) => {
```

Comment:
All `map()` and `filter()` calls produce new reference on every call. How can we improve performance of this component?